requirements.txt

```diff
@@ -1,2 +1,5 @@
 torch>=1.7
 numpy>=1.17
+scikit-image>=0.14.2
```

Comment:
Removed.

Reply:
If I remove the scikit-image from here, then the tests fail, is there any other alternative?

Reply:
Please add your new test to here: https://github.com/Project-MONAI/MONAI/blob/dev/tests/min_tests.py#L170
It will ignore the min tests if your new test has any 3rd dependencies.

Thanks.